### DISCUSSION TITLE
Way to automatically create bars on every monitor?

### DISCUSSION THREAD

**Discussion by BBaoVanC** [elkowar/eww#1166] [Q&A]
I'm used to using Waybar, where it automatically puts my bar on every monitor, including if I add new ones. Is there a way to do this with eww? I want to avoid hardcoding a set number of monitors and their layout in eww config and instead have it be dynamic based on the monitors that exist.

**Comment by w-lfchen** [comment]
to my knowledge, eww doesn't listen for monitor changes, so i'd recommend writing a script for that
